js/ui/windowManager.js

```diff
@@ -1458,7 +1456,7 @@ const WindowManager = new Lang.Class({
             }
         }
 
-        if (this._codingManager) {
+        if (global.settings.get_boolean('enable-behind-the-screen')) {
```

Comment:
What happens when the user disables `enable-behind-the-screen` and they still have some windows flipped over to builder windows? Do we try and reset all the state back to what the system would be like if the feature was disabled? Or should we do that in a later PR?